org.mwc.debrief.core/src/org/mwc/debrief/core/dialogs/ImportNMEADialog.java

```diff
@@ -0,0 +1,176 @@
+package org.mwc.debrief.core.dialogs;
+
+import org.eclipse.jface.dialogs.IDialogConstants;
+import org.eclipse.jface.dialogs.TitleAreaDialog;
+import org.eclipse.jface.viewers.ArrayContentProvider;
+import org.eclipse.jface.viewers.ColumnLabelProvider;
+import org.eclipse.jface.viewers.ComboViewer;
+import org.eclipse.jface.viewers.IBaseLabelProvider;
+import org.eclipse.jface.viewers.ISelectionChangedListener;
+import org.eclipse.jface.viewers.IStructuredSelection;
+import org.eclipse.jface.viewers.SelectionChangedEvent;
+import org.eclipse.jface.viewers.StructuredSelection;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.swt.widgets.Display;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.swt.widgets.Shell;
+import org.mwc.cmap.core.CorePlugin;
+
+public class ImportNMEADialog extends TitleAreaDialog
+{
+
+  /**
+   * @param args
+   */
+  public static void main(final String[] args)
+  {
+
+    Display display = Display.getDefault();
+    if (display.isDisposed())
+      display = new Display();
+    final Shell shell = new Shell(display, SWT.NO_TRIM);
+    final ImportNMEADialog dialog = new ImportNMEADialog(shell);
+    dialog.open();
+  }
+
+  private long ownshipFreq;
+
+  private long thirdPartyFreq;
+
+  public ImportNMEADialog()
+  {
+    this(Display.getDefault().getActiveShell());
+  }
+
+  public ImportNMEADialog(final Shell parentShell)
+  {
+    super(parentShell);
+  }
+
+  @Override
+  public void create()
+  {
+    super.create();
+    setTitle("Import NMEA Data");
+    getButton(IDialogConstants.OK_ID).setText("Import");
+    setTitleImage(CorePlugin.extendedGetImageFromRegistry("icons/48/NMEA.png"));
+    setMessage("Please choose the frequency at which data will be imported (or 'none' to not import that type).");
+  }
+
+  @Override
+  protected Control createDialogArea(final Composite parent)
+  {
+    final Composite base = (Composite) super.createDialogArea(parent);
+    final Composite composite = new Composite(base, SWT.NONE);
+    composite.setLayoutData(new GridData(GridData.FILL_BOTH
+        | GridData.GRAB_HORIZONTAL | GridData.GRAB_VERTICAL));
+    composite.setLayout(new GridLayout(2, false));
```

Comment:
@IanMayo  I is little tick in java to get scope inside method. so no need to worry about method name duplicate or use wrong variable